quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -1550,4 +1555,21 @@ function getFileOverwritePermission() {
   fi
 }
 
+function createAPIService {
+  zitiLogin
+  # Create the bind and dial configs
+  "${ZITI_BIN_DIR-}/ziti" edge create config ziti.rest.dial intercept.v1 '{"protocols":["tcp"],"addresses":["ziti.rest.api"], "portRanges":[{"low":443, "high":443}]}' > /dev/null
+  "${ZITI_BIN_DIR-}/ziti" edge create config ziti.rest.bind host.v1 '{"protocol":"tcp", "address":"'${ZITI_EDGE_CONTROLLER_HOSTNAME}'","port":'"${ZITI_EDGE_CONTROLLER_PORT}"'}' > /dev/null
+
+  # Create the service, applying the configs
+  "${ZITI_BIN_DIR-}/ziti" edge create service ziti.rest.service --configs "ziti.rest.bind,ziti.rest.dial" > /dev/null
+
+  # Create the service policies allowing dial and bind access
+  "${ZITI_BIN_DIR-}/ziti" edge create service-policy ziti.rest.service.bind Bind --service-roles "@ziti.rest.service" --identity-roles "#ziti.rest.host" > /dev/null
```

Comment:
what about ziti.rest.host --> ziti.rest.binders or something like that? same for dialers instead of `quickstart.rest.user`.. `ziti.rest.dialers`?